obspy/taup/tau.py

```diff
@@ -519,7 +519,7 @@ def get_pierce_points_geo(self, source_depth_in_km, source_latitude_in_deg,
                   "Arrivals object will not be modified. " + \
                   "Install the Python module 'geographiclib' to solve " + \
                   "this issue."
-            raise ImportError(msg)
+            warnings.warn(msg)
```

Comment:
In my opinion, this should raise a true Error. If someone calls the geo routine, he wants the geographical points. Otherwise he could just use the other routine. Another fallback should be somewhere else.